


def get_summ(a,b):
    try:
        return int(a) + int(b)

    except ValueError :
        print ("Некорректное значение")
print(get_summ('five',5))




    
